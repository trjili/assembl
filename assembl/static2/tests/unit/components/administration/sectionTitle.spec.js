import React from 'react';
import renderer from 'react-test-renderer';

import SectionTitle from '../../../../js/app/components/administration/sectionTitle';

describe('SectionTitle component', function () {
  it('should render an admin section title', function () {
    const i18n = {
      locale: 'fr',
      translations: {
        en: {
          administration: {
            survey: {
              0: 'My section title'
            }
          }
        },
        fr: {
          administration: {
            survey: {
              0: 'Mon titre de section'
            }
          }
        }
      }
    }
    const component = renderer.create(
      <SectionTitle
        i18n={i18n}
        phase="survey"
        tabId="0"
        annotation="foobar"
      />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})