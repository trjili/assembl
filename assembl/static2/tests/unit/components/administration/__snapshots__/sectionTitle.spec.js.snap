exports[`SectionTitle component should render an admin section title 1`] = `
<div>
  <h3
    className="dark-title-3">
    Mon titre de section
  </h3>
  <div
    className="box-hyphen" />
  <div
    className="annotation">
    foobar
  </div>
</div>
`;
